setup.php

```diff
@@ -73,22 +73,7 @@ function plugin_version_sccm() {
          'glpi'   => [
             'min' => PLUGIN_SCCM_MIN_GLPI,
             'max' => PLUGIN_SCCM_MAX_GLPI,
-            'plugins' => [
-               'fusioninventory',
-            ],
          ],
-         'php'    => [
-            'exts'=> [
-               'sqlsrv'    => [
-                  'required'  => true,
-                  'function'  => 'sqlsrv_connect'
-               ],
-               'curl'      => [
-                  'required'  => true,
-                  'function'  => 'curl_init'
-               ]
-            ]
-         ]
```

Comment:
Are you sure these extensions are no longer used ?

Reply:
sorry, is for test